plugins/formattingFilters/dateAndTime/DateTimeFilters.ts

```diff
@@ -0,0 +1,40 @@
+import Vue from 'vue'
+import VueMoment from 'vue-moment'
+import moment from 'moment'
+
+Vue.use(VueMoment, { moment })
+
+export class DateTimeFilters {
+    formatMilliToDateTime(value) {
+        if (value == null) {
```

Comment:
Avoid use of ==, if you need to check for undefined or null, it is better to check for them with strict equality to improve readability. Especially since this project is public